### ISSUE TITLE
[location] [TGN] preferred term vs. English term

### ISSUE THREAD

**Issue by SimonGruening** [exc-asia-and-europe/ziziphus#378]
I am not sure if it makes any sense to search for the preferred term in TGN as Getty seems to have set the vernacular name of a country or a place as preferred. I had difficulties finding e.g. Thailand today as I would have had to search for "Prathet Thai" instead of Thailand to get directed to the right TGN entry.

**Comment by MatthiasArnold** [comment]
search should include alternate names. 

for other names for thailand cf. http://www.getty.edu/vow/TGNFullDisplay?find=thailand&place=&nation=&prev_page=1&english=Y&subjectid=1000142

**Comment by SimonGruening** [comment]
Yeah, but it doesn't make any sense - though it is the preferred term in TGN - that only "Prathet Thai" is shown in the list, because "the eff" do I know the vernacular name of Thailand. ;)

![image](https://cloud.githubusercontent.com/assets/3437957/13492956/5079b3a6-e13b-11e5-9ce4-9fc0fd4cc693.png)
